src/wp-content/themes/twentytwenty/functions.php

```diff
@@ -235,6 +235,7 @@ function twentytwenty_register_scripts() {
  * thus it does not warrant having an entire dedicated blocking script being loaded.
  *
  * @since Twenty Twenty 1.0
+ * @deprecated Twenty Twenty 2.3 Disabled printing script by default.
```

Comment:
```suggestion
 * @deprecated Twenty Twenty 2.3 Removed from wp_print_footer_scripts action.
```

Reply:
I liked [commenting out the resource hints filters](https://github.com/WordPress/wordpress-develop/pull/2920#discussion_r1094648615) on [Trac 55985](https://core.trac.wordpress.org/ticket/55985) because it seemed self-documenting (with the priority and args parameters). But your suggestions mention the appropriate action, which should be enough.

Reply:
I don't feel strongly either way.

Reply:
Except, commented-out code would be [flagged by the `WordPress-Extra` ruleset](https://github.com/WordPress/WordPress-Coding-Standards/blob/fca9d9ef2dcd042658ccb9df16552f5048e7bb04/WordPress-Extra/ruleset.xml#L142-L148) for PHPCS. So I think it's better to not leave the commented-out code in place.